### ISSUE TITLE
Update JS wrapper with latest changes

### ISSUE THREAD

**Issue by TimoGlastra** [openwallet-foundation/askar#164]
There are changes to the API in #159.

Not sure if there's any other changes to the public api. @andrewwhitehead are there any other prs that introduced chagnes to the ffi interface?

**Comment by andrewwhitehead** [comment]
No, I think the only other change is the addition of the P-384 key type.
